iModelCore/ECDb/ECDb/ECDbSqlFunctions.h

```diff
@@ -219,7 +219,7 @@ struct PropExistsFunc final : ScalarFunction , ECDb::IECDbCacheClearListener {
     public:
         explicit PropExistsFunc(ECDbCR ecdb) : ScalarFunction(SQLFUNC_PropExists, 2, DbValueType::IntegerVal), m_ecdb(ecdb) {
             const_cast<ECDbR>(m_ecdb).AddECDbCacheClearListener(*this);
-            m_propMap.Build(m_ecdb, true);
+            // m_propMap.Build(m_ecdb, true);

```

Comment:
remove this commented out code and replace it with an explanation that it can't be done here and why.

Reply:
done